ci-operator/step-registry/openshift/microshift/e2e/footprint-and-performance/openshift-microshift-e2e-footprint-and-performance-commands.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+set -xeuo pipefail
+
+IP_ADDRESS="$(cat "${SHARED_DIR}"/public_address)"
+HOST_USER="$(cat "${SHARED_DIR}"/ssh_user)"
+INSTANCE_PREFIX="${HOST_USER}@${IP_ADDRESS}"
+
+echo "Using Host $IP_ADDRESS"
+
+mkdir -p "${HOME}/.ssh"
+cat <<EOF >"${HOME}/.ssh/config"
+Host ${IP_ADDRESS}
+  IdentityFile ${CLUSTER_PROFILE_DIR}/ssh-privatekey
+  StrictHostKeyChecking accept-new
+  ServerAliveInterval 30
+  ServerAliveCountMax 1200
+EOF
+chmod 0600 "${HOME}/.ssh/config"
+
+cat << 'EOF' > /tmp/prepare.sh
+#!/bin/bash
+set -xeuo pipefail
+
+if ! sudo subscription-manager status >&/dev/null; then
+    sudo subscription-manager register \
+        --org="$(cat /tmp/subscription-manager-org)" \
+        --activationkey="$(cat /tmp/subscription-manager-act-key)"
+fi
+
+cp /tmp/pull-secret "${HOME}/.pull-secret.json"
+
+mkdir -p -m 0700 ${HOME}/.aws/
+
+# Profile configuration
+cat <<EOF2 >> ${HOME}/.aws/config
+[microshift-ci]
+region = us-west-2
+output = json
+EOF2
+
+# Profile credentials
+cat <<EOF2 >>${HOME}/.aws/credentials
+[microshift-ci]
+aws_access_key_id = $(cat /tmp/aws_access_key_id)
+aws_secret_access_key = $(cat /tmp/aws_secret_access_key)
+EOF2
+
+# Permissions and environment settings
+chmod -R go-rwx ${HOME}/.aws/
+
+chmod 0755 ~
+tar -xf /tmp/microshift.tgz -C ~ --strip-components 4
+EOF
+chmod +x /tmp/prepare.sh
+
+tar czf /tmp/microshift.tgz /go/src/github.com/openshift/microshift
+scp \
+  /tmp/prepare.sh \
+  /var/run/rhsm/subscription-manager-org \
+  /var/run/rhsm/subscription-manager-act-key \
+  /var/run/microshift-dev-access-keys/aws_access_key_id \
+  /var/run/microshift-dev-access-keys/aws_secret_access_key \
+  "${CLUSTER_PROFILE_DIR}/pull-secret" \
+  /tmp/microshift.tgz \
+  "${INSTANCE_PREFIX}:/tmp"
+
+ssh "${INSTANCE_PREFIX}" "/tmp/prepare.sh"
+
+ssh "${INSTANCE_PREFIX}" 'bash -x $HOME/microshift/scripts/ci-footprint-and-performance/1-setup.sh'
```

Comment:
Since the host is rebooted in the end, it might return error code. Should we just add `|| true` to ssh command and assume setup always succeeds? We need a comment here in any case about the reboot.

Reply:
Maybe, we should run reboot in a separate ssh invocation?